python/GafferUI/PathChooserWidget.py

```diff
@@ -49,6 +49,8 @@ def __init__( self, path, previewTypes=[], allowMultipleSelection=False, bookmar
 
 		self.__column = GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Vertical, spacing=8 )
 
+		self.__column._qtWidget().setObjectName( "gafferPathChooserWidget" )
```

Comment:
I had assumed that we'd be doing this in `Widget.__init__`, editing `self._qtWidget()`, so the style could use the same trick for any widget type (and probably using a property rather than the object name, since I think we already use the object name other places). Is there a reason that won't work?

Reply:
Oh thats a much better idea.

Reply:
How about https://github.com/GafferHQ/gaffer/pull/3229/commits/1b3612d486f58a24c27e3b1820b778e23ee97deb (note, the order that gitub lists commits doesn't reflect a re-arrange in rebase)